src/components/stepforms/ConditionsEditor/ConditionsGroup.vue

```diff
@@ -194,6 +192,33 @@ export default class ConditionsGroup extends Vue {
     return groupIndex === this.groups.length - 1;
   }
 
+  async resetOperatorIfNecessary() {
+    // await the re-render to received the updated tree (emitted + received in props)
+    await this.$nextTick();
+
+    // reset the operator to an empty string when there's only one condition left
+    if (this.conditions.length === 1 && this.groups.length === 0) {
+      const newConditionsTree = {
+        ...this.conditionsTree,
+        operator: '',
+      } as AbstractFilterTree;
+      this.emitUpdatedConditionTree(newConditionsTree);
+    }
+    return;
+  }
+
+  setOperatorIfNecessaryAndUpdateConditionTree(newConditionsTree: AbstractFilterTree) {
```

Comment:
```suggestion
  updateConditionTree(newConditionsTree: AbstractFilterTree) {
```
Only `updateConditionTree` is better I think

Reply:
ok :)